packages/ember-metal/lib/property_set.js

```diff
@@ -62,8 +62,11 @@ export function set(obj, keyName, value, tolerant) {
     return setPath(obj, keyName, value, tolerant);
   }
 
-  assert(`calling set on destroyed object: ${toString(obj)}.${keyName} = ${toString(value)}`,
-         !obj.isDestroyed);
+  if (obj.isDestroyed) {
```

Comment:
@krisselden should we actually allow setting through destroyed objects or to the descs? My intuition leads me to believe this should actually occur before line 53.